index.html

```diff
@@ -43,6 +43,139 @@ <h3 class="title dtitle">I'm a software developer.</h3>
+                            <li class="card-tool">Ruby on rails</li>
+                            <li class="card-tool"> CSS</li>
+                            <li class="card-tool">Javascript</li>
+                            <li class="card-tool"> HTML</li>
+                        </ul>
+                        <button type="button" class="see-project btn">See project</button>
+                    </div>
+                </div>
+                <div class="card">
+                    <div class="card-image"></div>
+                    <div class="card-text">
+                        <p class="card-title">Multi-Post Stories Gain + Glory</p>
+                        <ul class="card-tools-list">
+                            <li class="card-tool">Ruby on rails</li>
+                            <li class="card-tool"> CSS</li>
+                            <li class="card-tool">Javascript</li>
+                            <li class="card-tool"> HTML</li>
+                        </ul>
+                        <button type="button" class="see-project btn">See project</button>
+                    </div>
+                </div>
+                <div class="card">
+                    <div class="card-image"></div>
+                    <div class="card-text">
+                        <p class="card-title">Multi-Post Stories Gain + Glory</p>
+                        <ul class="card-tools-list">
+                            <li class="card-tool">Ruby on rails</li>
+                            <li class="card-tool"> CSS</li>
+                            <li class="card-tool">Javascript</li>
+                            <li class="card-tool"> HTML</li>
+                        </ul>
+                        <button type="button" class="see-project btn">See project</button>
+                    </div>
+                </div>
+                <div class="card">
+                    <div class="card-image"></div>
+                    <div class="card-text">
+                        <p class="card-title">Multi-Post Stories Gain + Glory</p>
+                        <ul class="card-tools-list">
+                            <li class="card-tool">Ruby on rails</li>
+                            <li class="card-tool"> CSS</li>
+                            <li class="card-tool">Javascript</li>
+                            <li class="card-tool"> HTML</li>
+                        </ul>
+                        <button type="button" class="see-project btn">See project</button>
+                    </div>
+                </div>
+                
+            </div>
+        </section>
+
+        <section class="about-me">
+            <div class="text">
+                <h2 class="primary-text">About me</h2>
+                <div class="supporting-text"> I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, 
+                    don’t hesitate to contact me.</div>
```

Comment:
- [ ] **[OPTIONAL]** Instead of using a `div` for your paragraph, consider using a `p` tag. This will not only help you follow best `HTML` practices but also make your code more organized and easier to read. 🙌